### ISSUE TITLE
Header Mapping doesn't work as expected.

### ISSUE THREAD

**Issue by KarachentsevDanil** [JamesRandall/FunctionMonkey#33]
Header Mapping doesn't work as expected. 
I want to bind my Authorization header to command property, in order to do this I use the following construction.
  .HttpRoute("/Invoice", route => route
                    .HttpFunction<MyCommand>(HttpMethod.Get)
                        .Options(options => options.AddHeaderMapping(x => x.Authorization, "Authorization"))
                )

And this Authorization property always null.

Do you have any suggestions on why it can not work?

**Comment by AKomyshan** [comment]
Hi James, Hi Danil. I did a little research on this issue.
TR;DR the problem in `http.csharp.handlebars` file or in `HeaderBindingConfiguration` class.

The condition is no longer relevant most likely
`{{#if HeaderBindingConfiguration.Enabled}}` because `Enabled` property no longer exists.

I revised the commit [9f718e4](https://github.com/JamesRandall/FunctionMonkey/commit/9f718e44d84a3d1cbad14a0ed022e74daaa393e5) and as we can see, the `Enabled` property has been removed from the `HeaderBindingConfiguration`class, but `http.csharp.handlebars` file still has a check of this property.

I can suggest to return this property for example as follows:
`public bool Enabled => PropertyFromHeaderMappings.Any();`
i don't know how Handlebars and Mustache works under the hood but maybe this can have a positive impact on the performance.

Or just simply remove `{{#if HeaderBindingConfiguration.Enabled}}` from template because `{{#if HeaderBindingConfiguration}}` true only when the `HttpFunctionOptionsBuilder<TCommand>.AddHeaderMapping` was called.

Hopefully this helps
